### ISSUE TITLE
must run onSuccess on pre Build.VERSION_CODES.M

### ISSUE THREAD

**Issue by DJafari** [NoNews/NoPermission#1]
i think you must call `successListener.onSuccess();` on pre  `Build.VERSION_CODES.M`

https://github.com/NoNews/NoPermission/blob/master/library/src/main/java/ru/alexbykov/nopermission/PermissionHelper.java#L58

```java
public void run() {
        if (isNeedAskPermission()) {

            if (successListener != null && failureListener != null && newerAskAgainListener != null) {
                activity.requestPermissions(permissions, PERMISSION_REQUEST_CODE);
            } else
                throw new RuntimeException("OnPermissionSuccessListener or OnPermissionFailureListener not implemented. Use methods: onSuccess and onFailure");
        } else if(successListener != null) {
                successListener.onSuccess();
        }
    }
```

**Comment by NoNews** [comment]
Hi @DJafari, thanks for this issue, will fix it on 1.0.8

**Comment by DJafari** [comment]
@NoNews i send a pull request to fix some problems and improvments

**Comment by NoNews** [comment]
@DJafari Thank you, I'll review and merge today.
Just too I am engaged in refactoring and correction of small bugs, I will try to lay out 1.8 as soon as possible

**Comment by NoNews** [comment]
fixed in 1.8
